code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -0,0 +1,182 @@
+#define ETHEREAL_COLORS list("#00ffff", "#ffc0cb", "#9400D3", "#4B0082", "#0000FF", "#00FF00", "#FFFF00", "#FF7F00", "#FF0000")
+
+/datum/species/ethereal
+	name = "Ethereal"
+	id = "ethereal"
+	attack_verb = "burn"
+	attack_sound = 'sound/weapons/etherealhit.ogg'
+	miss_sound = 'sound/weapons/etherealmiss.ogg'
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/ethereal
+	mutantstomach = /obj/item/organ/stomach/ethereal
+	mutanttongue = /obj/item/organ/tongue/ethereal
+	exotic_blood = /datum/reagent/consumable/liquidelectricity //Liquid Electricity. fuck you think of something better gamer
```

Comment:
this is uh way too strong, change it to a weakened subtype or something not as powergamey

Reply:
nerf it when needed tbh